pkg/metrics/timer/timer.go

```diff
@@ -0,0 +1,76 @@
+package timer
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Timer is the struct that keeps track of each of the sections.
+type Timer struct {
+	startTime    time.Time
+	totalTime    time.Time
+	listOfStages []string
+	stageTimes   map[string]time.Duration
+}
+
+const (
+	// InfrastructureStageName is a constant string value that is used to display the Infrastructure created stage text in the logs.
+	InfrastructureStageName = "Infrastructure"
+	// APIStageName is a constant string value that is used to display the API stage text in the logs.
+	APIStageName = "API"
+	// BootstrapCompleteStageName is a constant string value that is used to display the Bootstrap Complete text in the logs.
+	BootstrapCompleteStageName = "Bootstrap Complete"
+	// BootstrapDestroyStageName is a constant string value that is used to display the Boostrap Destroy Complete text in the logs.
+	BootstrapDestroyStageName = "Bootstrap Destroy"
+	// ClusterOperatorStageName is a constant string value that is used to display the Cluster Operator Complete text in the logs.
+	ClusterOperatorStageName = "Cluster Operator"
+
+	// ConsoleStageName is a constant string value that is used to display the Cluster Operator Complete text in the logs.
+	ConsoleStageName = "Console"
```

Comment:
why does the timer package get to control what stages can exist?

Reply:
These are just constants to use. I've kept them here as they were relevant to the timer.

Reply:
I want this package to not provide these const. I think this is un necessarily restricts the users with what stages they can use.

Reply:
I think the only one I think that can should stay here is Total